packages/erxes-ui/src/components/richTextEditor/extensions/Mention.ts

```diff
@@ -1,26 +1,26 @@
-import Mention from '@tiptap/extension-mention';
+import TiptapMention from '@tiptap/extension-mention';
 import { mergeAttributes } from '@tiptap/core';
 
-export const MentionExtended = Mention.extend({
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **🚨 issue (security):** Ensure that the exported name aligns with the imported module name for clarity.